element-hq/element-x-android
# Optimize media upload

## Content


- Wording update
- Enable optimization by default
- Always limit video size
- Compress png to png

## Motivation and context


https://github.com/element-hq/element-meta/issues/2543

## Screenshots / GIFs


See recorded ones

## Tests



- Enable or disable the new setting
- Send media to the room and observe that it is compressed when it needs to

## Tested devices

- OS version(s):

## Checklist
